Sources/LiveViewNative/Modifiers/View Configuration Modifiers/DraggableModifier.swift

```diff
@@ -45,16 +50,12 @@ struct DraggableModifier<R: RootRegistry>: ViewModifier, Decodable {
     func body(content: Content) -> some View {
         #if !os(watchOS)
```

Comment:
if it doesn't support tvOS this could be used:
```suggestion
        #if os(iOS) || os(macOS)
```